README.md

```diff
@@ -768,10 +773,10 @@ Joins a gateway to the ar.io network via its associated wallet. Requires `signer
 
 ```typescript
 const jointNetworkParams = {
```

Comment:
This has a typo that could be fixed to "joinNetworkParams"

Reply:
[5d7423a](https://github.com/ar-io/ar-io-sdk/pull/84/commits/5d7423a0a81ea3412c04497644c478a0670ca433)